common/views/components/FeaturedCard/FeaturedCard.js

```diff
@@ -31,6 +32,20 @@ type Props = {|
   isReversed?: boolean,
 |};
 
+export function convertCardToFeaturedCardProps(item: Card) {
```

Comment:
What type is this returning?

Reply:
It's partial `FeaturedCard` Props (without `children` and a number of props that are determined by the template – e.g. background colour determined by position in list).

Reply:
Could we put this in the type system then? `PartialFeatureCard` sounds good?